DSCResources/MSFT_xDatabaseLogin/MSFT_xDatabaseLogin.schema.mof

```diff
@@ -5,7 +5,6 @@ class MSFT_xDatabaseLogin : OMI_BaseResource
     [Write, ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
     [Key] String LoginName;
     [Write] String LoginPassword;
-    [Required, ValueMap{"Windows","SQL"}, Values{"Windows","SQL"}] string SqlAuthType;
```

Comment:
This is a breaking change.

Really, we don't want to support Integrated/Windows Auth?  This is the recommended auth for security.